front/app/components/UI/UserName/messages.ts

```diff
@@ -13,4 +13,8 @@ export default defineMessages({
     id: 'app.components.UserName.anonymous',
     defaultMessage: 'Anonymous',
   },
+  anonymousTooltip: {
+    id: 'app.components.UserName.anonymousTooltip',
+    defaultMessage: 'This user has decided to anonymise their contribution',
```

Comment:
Not sure whether our default english locale using UK or US english. If the latter, then this will need to be anonymize I think

Reply:
Good point - I think it's US actually. I'll change it :) Thanks!!